docker/test/nginx.Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM nginx
```

Comment:
What version should we pin here?

Reply:
You could use the latest working version provided by docker hub. That can be our baseline.

Reply:
Makes sense. I have now tagged this as "1.23.1" as that is in line with "latest" at the time of writing. Unless I should just use "latest".

Reply:
Using the actual version number like you've done here is what I meant.

You can mark this as resolved.